base/ntuple.jl

```diff
@@ -42,6 +42,23 @@ ntuple(f, ::Val{1}) = (@_inline_meta; (f(1),))
 ntuple(f, ::Val{2}) = (@_inline_meta; (f(1), f(2)))
 ntuple(f, ::Val{3}) = (@_inline_meta; (f(1), f(2), f(3)))
 
+"""
+    ntuple(f::Function, ::Val{N})
```

Comment:
```suggestion
    ntuple(f, ::Val{N})
```

It allows arbitrary callables (not just a `Function`). So let's reflect it to the docstring.